src/game/level/lex.rs

```diff
@@ -33,7 +33,7 @@ pub enum LexError {
 }
 
 pub fn parse(raw_data: &str) -> impl Iterator<Item = Result<(usize, RawStatement), LexError>> {
-	let assignment_regex = Regex::new(r"^(?<KEY>[a-zA-Z0-9_]+)=(?<VALUE>.+)$")
+	let assignment_regex = Regex::new(r"^(?<KEY>[a-zA-Z0-9_]+)=(?<VALUE>.*)$")
```

Comment:
Empty assignments are then valid?
Such as:
```
A=
```

Reply:
Such was the intention. Did you mean to explicitly prevent it?

Reply:
I don't have anything directly against it, it just seemed like a malformed statement in all scenarios I considered at the time.

Reply:
My problem with this was that before this, the empty assignment actually resolved to an invalid keyword error somehow

Reply:
I generally didn't want to impose any limitations, since multiple equal signs are not really a problem (the first sign is the assignment), and we are already limiting ourselfs to ascii earlier in the code.

I guess empty string assignments are also valid.